style.css

```diff
@@ -10,12 +7,22 @@
 }
 
 body {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Open Sans",
+    "Helvetica Neue",
+    sans-serif;
   line-height: 1.5;
   font-size: 0.875rem;
-  min-height: 100vh;
-  max-width: 100vw;
+  width: 375px;
```

Comment:
Great work, but doing so is breaking the layout of your application. As you can see, there is a huge white spacing issue on the right side of your application. You can either remove it or go back to the previous code.
![image](https://user-images.githubusercontent.com/33268927/186376146-de437278-9345-48f4-9910-615966e6a62a.png)